tests/schema/extensions/test_opentracing.py

```diff
@@ -0,0 +1,133 @@
+import pytest
```

Comment:
I think we might need to restructure these tests to check what info we are sending to the tracer 😊

Reply:
I hope you can guide me on that